settings.py

```diff
@@ -752,7 +752,7 @@ def _color_scheme_completions(view):
                 "{}  \tPackage: {}".format(file_name, package),
                 scheme_path
             )
-            completions.add(item)
+            completions.append(item)
         return completions
```

Comment:
Sorting completions on return of `value_completions()` is just enough. Why sorting twice? You just should not return a set() here which may be accessed by index later in `value_completions()` .

With a second look into `value_completions()` ...
1. I find it a bit messy (even pylint complains about too much branches) and 
2. The whole stuff beginning with line 630 is useless to apply to color scheme / theme completions. It just needs to apply to comments read from comments or the "default" completion.